runtime/lua/vim/lsp/log.lua

```diff
@@ -32,6 +32,17 @@ do
 
   vim.fn.mkdir(vim.fn.stdpath('cache'), "p")
   local logfile = assert(io.open(logfilename, "a+"))
+
+	local log_info = vim.loop.fs_stat(logfilename)
+	if log_info and log_info.size > 1e9 then
+		local warn_msg = string.format(
+			"LSP client log is large (%d MB): %s",
+			log_info.size / (1000 * 1000),
```

Comment:
indentation :O

Reply:
Oh, I thought it was the indentation for which I accepted your GH edit, my bad. I'll fix.

Reply:
looking at this I am unclear which part of the indentation you are referring to

Reply:
we use 2-space indent everywhere, and also the entire block starts at the wrong column